### PR TITLE
feat: Add hover effect to footer anchor tags

### PR DESCRIPTION
**Pull request by lakshmishreea122003**
The anchor tags in footer get underlined when hovered.
Screenshot 2024-02-20 191150

### CODE REVIEW

**Comment by MarkEWaite** [comment]
Please provide a "before" and "after" screenshot so that reviewers understand what you were trying to change.

Please provide more details of the tests that you performed to confirm that the change did not have unexpected side effects.

**Comment by halkeye** [comment]
> Please provide a "before" and "after" screenshot so that reviewers understand what you were trying to change.

And the title please, "update anchor" isn't a changelog message

also @zbynek did we not add light/darkmode into storybook? I don't see an option on https://deploy-preview-135--jenkins-io-components.netlify.app/?path=/story/example-footer--default

**Comment by zbynek** [comment]
@halkeye currently the dark/light/auto setting only works for Navbar stories, not footer (because footer is always white text + the same shade of blue...).

**Comment by lakshmishreea122003** [comment]
On hovering the anchor tags get underlined.
Screenshot 2024-02-21 181842

**Comment by lakshmishreea122003** [comment]
Screenshot 2024-02-21 182344

**Comment by lakshmishreea122003** [comment]
Should I make any changes or are there any corrections to be done? Please do let me know.

**Comment by zbynek** [comment]
There is no point setting the color, it is already white. Otherwise 👍

File: src/jio-footer.css [hunk 1]
```diff
@@ -44,6 +44,11 @@ a {
   text-underline-offset: 2px;
 }
 
+a:hover {
+  text-decoration: underline; 
+  color: #fff;
```
Comment (lemeurherve) [inline src/jio-footer.css:49]:
```suggestion
```
Already set

**Review by lemeurherve — DISMISSED** [review]

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**